Website/src/rendering/src/assets/css/layout/section-speakers.css

```diff
@@ -1,3 +1,47 @@
+.featured-speakers {
```

Comment:
This should go in a `/components/FeaturedSpeakers.css` file as it related to the FeaturedSpeaker component, not the speakers section of the home page.